core/tests/test_utils.py

```diff
@@ -231,15 +289,43 @@ def _parse_json_response(self, json_response, expect_errors=False):
         return json.loads(json_response.body[len(feconf.XSSI_PREFIX):])
 
     def get_json(self, url, params=None, expect_errors=False):
-        """Get a JSON response, transformed to a Python object."""
+        """Get a JSON response, transformed to a Python object.
+
+        Args:
+            url: str. A URL for the GET request.
+            params: dict. The names and values of the parameters to be passed
+            in the GET call.
+            expect_errors: bool. If False, then if anything is written
+            to wsgi.errors (module webtest) will be an error.
+
+        Returns:
+            dict. The GET response.
+        """
         json_response = self.testapp.get(
             url, params, expect_errors=expect_errors)
         return self._parse_json_response(
             json_response, expect_errors=expect_errors)
 
     def post_json(self, url, payload, csrf_token=None, expect_errors=False,
                   expected_status_int=200, upload_files=None):
-        """Post an object to the server by JSON; return the received object."""
+        """Post an object to the server by JSON; return the received object.
+
+        Args:
+            url: str. The URL to make a POST request to.
+            payload: dict (json). data sent to the server to make
+            the post request.
```

Comment:
Indent continuation line by 4, here and in the rest of the args list. 
post --> POST, here and below.